attributes/default.rb

```diff
@@ -16,3 +16,5 @@
 default['onlinefs']['hopsworks']['password']  = "onlinefspw"
 
 default['onlinefs']['monitoring']             = 12800
+
+default['onlinefs']['service']['thread_number'] = 10
```

Comment:
add this in the `metadata.rb` otherwise is not going to be configurable from the cluster definition.

Reply:
see hopsworks-chef for an example on how to do it.